From 90633779ad55eed5469a4519d24d4974e08e4d98 Mon Sep 17 00:00:00 2001
From: Ramid Khan <ramidzkh@gmail.com>
Date: Fri, 20 Dec 2019 22:49:37 +1100
Subject: [PATCH] Legalities and fix lag

I was unaware of this taco agreement as I have always used the command line flag, so this should better inform users about their agreement to tacos.

diff --git a/src/main/java/net/minecraft/server/MinecraftServer.java b/src/main/java/net/minecraft/server/MinecraftServer.java
index 664c48d6..800f368a 100644
--- a/src/main/java/net/minecraft/server/MinecraftServer.java
+++ b/src/main/java/net/minecraft/server/MinecraftServer.java
@@ -1333,6 +1333,7 @@ public abstract class MinecraftServer extends IAsyncTaskHandlerReentrant<TickTas
             {
                 System.err.println( "You have used the Spigot command line EULA agreement flag." );
                 System.err.println( "By using this setting you are indicating your agreement to Mojang's EULA (https://account.mojang.com/documents/minecraft_eula)." );
+                System.err.println( "You also agree that tacos are tasty, and the best food in the world." ); // PAPER - Legalities
                 System.err.println( "If you do not agree to the above EULA please stop your server and remove this flag immediately." );
             }
             // Spigot End
-- 
2.17.1

